airflow/www/templates/airflow/variable_edit.html

```diff
@@ -0,0 +1,29 @@
+{#
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+#}
+{% extends 'appbuilder/general/model/edit.html' %}
+
+{% block tail %}
+  {{ super() }}
+  <style>
+    #val { font-family: monospace; }
+  </style>
+  <script>
+    const height = Math.min(window.innerHeight * 0.5, val.scrollHeight);
+    document.getElementById('val').style.height = `${height}px`;
```

Comment:
@ashb this is embarrassing, but I just realized that I messed this up when I made the change for code review. It actually works as is (which is why I didn't catch it!), but only because I was "lucky" -- `val` is already defined in the global scope, from what I can tell.

I would like to change this script to:
```
    const val = document.getElementById('val');
    const height = Math.min(window.innerHeight * 0.5, val.scrollHeight);
    val.style.height = `${height}px`;
```
What's the best course of action here? Do I need to create a Jira ticket for this?

Reply:
@eeshugerman It hasn't been release yet so you should use the same Jira.
This is what I did when I fixed a bug created in first PR of https://issues.apache.org/jira/browse/AIRFLOW-4136

Reply:
Thanks @eladkal, I've added a new PR referencing the same jira ticket: https://github.com/apache/airflow/pull/5641